hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/WritableTokenRelationStore.java

```diff
@@ -62,6 +62,16 @@ public void put(@NonNull final TokenRelation tokenRelation) {
                         Objects.requireNonNull(tokenRelation));
     }
 
+    /**
+     * Removes a {@link TokenRelation} from the state
+     *
+     * @param tokenRelation the {@code TokenRelation} to be removed
+     */
+    public void remove(@NonNull final TokenRelation tokenRelation) {
+        requireNonNull(tokenRelState)
```

Comment:
Would it be better to do that check in the constructor?

Reply:
Fair question. I'm not sure what the answer is. The constructor instantiates `tokenRelState`, but I'm not sure if it should ever be null. I was following the pattern of the other methods in this class.